src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <h4>List of books</h4>
+    <li>
+      <button type="button">Remove</button>
+    </li>
```

Comment:
- [ ] This component should display a list of `Book` (example name) components, please create another component for each display a single book 😃